packages/babel-plugin-transform-block-scoping/test/fixtures/exec/scope-bindings.js

```diff
@@ -3,36 +3,35 @@ if (x) {
   const bar = 1;
 }`;
 
+var expected = `var foo = 1;
+
+if (x) {
+  var bar = 1;
+}`;
+
 var innerScope = true;
-var res = transform(code, {
+return transformAsync(code, {
```

Comment:
Can we move the `runCodeInThisContext` wrapper

https://github.com/babel/babel/blob/b1e73d6f961065c56427ffa89c130beea8321d3b/packages/babel-helper-transform-fixture-test-runner/src/index.ts#L228

before the Babel transform?

https://github.com/babel/babel/blob/b1e73d6f961065c56427ffa89c130beea8321d3b/packages/babel-helper-transform-fixture-test-runner/src/index.ts#L302

So that we don't have to change test options and `return` in `exec.js` is automatically valid.